packages/react/src/patterns/sections/CardSection/CardSection.js

```diff
@@ -1,12 +1,18 @@
-import { featureFlag, sameHeight } from '@carbon/ibmdotcom-utilities';
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
 import React, { useEffect, useRef } from 'react';
 import { ArrowRight20 } from '@carbon/icons-react';
 import { Card } from '../../sub-patterns/Card';
 import classNames from 'classnames';
-import { DDS_CARD_SECTION } from '../../../internal/FeatureFlags';
 import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
 import PropTypes from 'prop-types';
 import root from 'window-or-global';
+import { sameHeight } from '@carbon/ibmdotcom-utilities';
```

Comment:
@mkothur This can be imported from `L12`

Reply:
@kennylam what is L12?

Reply:
@kennylam I think this has been removed in this pull request.